content/docs/examples/multicluster/index.md

```diff
@@ -0,0 +1,312 @@
+    _Note:_ replace `mygcp@gmail.com` with the email tied to your Google Cloud account:
+
+    {{< text bash >}}
+    $ KUBE_USER="mygcp@gmail.com"
+    $ kubectl create clusterrolebinding gke-cluster-admin-binding \
+      --clusterrole=cluster-admin \
+      --user="${KUBE_USER}"
+    {{< /text >}}
+
+## Create a Google Cloud firewall rule
+
+The following rule needs to be created to allow each clusters' pods to directly communicate:
+
+{{< text bash >}}
+$ function join_by { local IFS="$1"; shift; echo "$*"; }
+$ ALL_CLUSTER_CIDRS=$(gcloud container clusters list --format='value(clusterIpv4Cidr)' | sort | uniq)
+$ ALL_CLUSTER_CIDRS=$(join_by , ${ALL_CLUSTER_CIDRS})
+$ ALL_CLUSTER_NETTAGS=$(gcloud compute instances list --format='value(tags.items.[0])' | sort | uniq)
+$ ALL_CLUSTER_NETTAGS=$(join_by , ${ALL_CLUSTER_NETTAGS})
+$ gcloud compute firewall-rules create istio-multicluster-test-pods \
+  --allow=tcp,udp,icmp,esp,ah,sctp \
+  --direction=INGRESS \
+  --priority=900 \
+  --source-ranges="${ALL_CLUSTER_CIDRS}" \
+  --target-tags="${ALL_CLUSTER_NETTAGS}" --quiet
+{{< /text >}}
+
+## Install the Istio control plane
+
+The following generates an Istio installation manifest, installs it, and enables automatic sidecar injection in
+the `default` namespace:
+
+{{< text bash >}}
+$ kubectl config use-context "gke_${proj}_${zone}_cluster-1"
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system > $HOME/istio_master.yaml
+$ kubectl create ns istio-system
+$ kubectl apply -f $HOME/istio_master.yaml
+$ kubectl label namespace default istio-injection=enabled
+{{< /text >}}
+
+Wait for pods to come up:
+
+{{< text bash >}}
+$ kubectl get pods -n istio-system
+{{< /text >}}
+
+## Generate remote cluster manifest
+
+1.  Get control plane pod IPs:
+
+    {{< text bash >}}
+    $ export PILOT_POD_IP=$(kubectl -n istio-system get pod -l istio=pilot -o jsonpath='{.items[0].status.podIP}')
+    $ export POLICY_POD_IP=$(kubectl -n istio-system get pod -l istio=mixer -o jsonpath='{.items[0].status.podIP}')
+    $ export STATSD_POD_IP=$(kubectl -n istio-system get pod -l istio=statsd-prom-bridge -o jsonpath='{.items[0].status.podIP}')
+    $ export TELEMETRY_POD_IP=$(kubectl -n istio-system get pod -l istio-mixer-type=telemetry -o jsonpath='{.items[0].status.podIP}')
+    {{< /text >}}
+
+1.  Generate remote cluster manifest:
+
+    {{< text bash >}}
+    $ helm template install/kubernetes/helm/istio-remote --namespace istio-system \
+      --name istio-remote \
+      --set global.remotePilotAddress=${PILOT_POD_IP} \
+      --set global.remotePolicyAddress=${POLICY_POD_IP} \
+      --set global.remoteTelemetryAddress=${TELEMETRY_POD_IP} \
+      --set global.proxy.envoyStatsd.enabled=true \
+      --set global.proxy.envoyStatsd.host=${STATSD_POD_IP} > $HOME/istio-remote.yaml
+    {{< /text >}}
+
+## Install remote cluster manifest
+
+The following installs the minimal Istio components and enables automatic sidecar injection on
```

Comment:
Change to active voice, fix the prepositions, and introduce the suggested numbered steps with:
To install the minimal Istio components and enable automatic sidecar injection in the `default` namespace on the remote cluster, follow these steps: